blog/creative-work.schema.yaml

```diff
@@ -9,11 +9,11 @@ properties:
       type: string
```

Comment:
Hi, please also update here to the new tag, or its list is empty.

![image](https://user-images.githubusercontent.com/2106987/192682089-49eebcd0-ac4a-43c4-aa02-bb88f70f0634.png)